.jenkins/Jenkinsfile-Update-Homebrew-OpenSSL

```diff
@@ -0,0 +1,35 @@
+def configs = ["sierra", "yosemite"]
+
+def build(label) {
+    node(label) {
+        try {
+            timeout(time: 30, unit: 'MINUTES') {
+                stage("Compile") {
+                    sh """
+                        set -xe
+
+                        /usr/local/bin/brew update
+                        /usr/local/bin/brew reinstall openssl@1.1 --build-bottle
```

Comment:
Only on platforms where there's no pre-built bottle. At the moment sierra and yosemite both have bottles, but when sierra+1 comes out yosemite bottles will stop being built so we'll need `--build-bottle` then to prevent it from building OpenSSL with CPU-specific optimizations.